src/util/types.ts

```diff
@@ -1679,6 +1679,7 @@ export interface SeriesLargeOptionMixin {
 export interface SeriesStackOptionMixin {
     stack?: string
     stackStrategy?: 'samesign' | 'all' | 'positive' | 'negative';
+    stackOrder?: 'normal' | 'reverse';
```

Comment:
Consider using the same value of [tooltip.order](https://echarts.apache.org/en/option.html#tooltip.order) like: `stackOrder?: 'seriesAsc' | 'seriesDesc'`.